src/main/java/uk/gov/hmcts/reform/blobrouter/config/EmailConfig.java

```diff
@@ -0,0 +1,38 @@
+package uk.gov.hmcts.reform.blobrouter.config;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Lazy;
+import uk.gov.hmcts.reform.blobrouter.services.email.MessageSender;
+
+import java.io.File;
+import java.util.Map;
+
+@Configuration
+@Lazy
```

Comment:
why lazy?

Reply:
It'll create the bean only when required

Reply:
if there is any error I think it is better to see them in loading .